src/main/resources/anet-schema.yml

```diff
@@ -552,4 +552,3 @@ properties:
         title: The list of ignored (email) domain names. These will not be affected by auto-inactivation or be sent email warnings.
         description: Valid email domain names for this ANET instance; may contain wildcards.
         examples: ["*.nato.int", dds.mil]
-
```

Comment:
Needs some more changes to the schema (`automaticallyInactivateUsers` may be optional, but it doesn't declare its type, and the sub-fields should be required *iff* it is defined). I'll push an update.